src/core/builtin_functions.cc

```diff
@@ -721,6 +721,25 @@ Value builtin_parent_module(Arguments arguments, const Location& loc)
   return {UserModule::stack_element(s - 1 - n)};
 }
 
+Value builtin_evalstring(const std::shared_ptr<const Context>& context, const FunctionCall * func_call)
+{
+  Arguments arguments(func_call->arguments, context);
+  Location loc = Location::NONE;
```

Comment:
`Location::NONE` looks odd to me.
Other build in functions have a  `const Location& loc` parameter.

LOC is used in check_arguments, so lets say I  pass the number 3.14 to the function, then check arguments will create a warning, but not say the location?

Reply:
Hi MichaelPFrey,

Thank you for code  review.
Yes i am am aware, that evalstring  does not use a valid Location for error debugging.
Issue is that the available function  prototypes which are valid arguents to new BuiltinFunction are very sparse.
The Function prototype needs access to the Context.
The Function prototype needs to  be able to return a Value
Optionally  the Function Prototype has access to the correct Location.

Unfortunately i did not find such a prototype, this is why I traded off the Location use use Location::None instead.

Actually this is same as 'is_undef' function,which does  not use Location either.

If you show me which function prototype  has all the stuff i need, I eager to  use  a good "Location" parameter.